app.py

```diff
@@ -1,11 +1,65 @@
-from flask import Flask
+from flask import Flask, render_template, url_for, redirect
+from flask_sqlalchemy import SQLAlchemy
+from flask_wtf import FlaskForm
+from wtforms import StringField, SubmitField
+from wtforms.validators import DataRequired
+import random
+import string
 
 app = Flask(__name__)
+app.config.from_pyfile("config.py")
+db = SQLAlchemy(app)
 
-print("Hello, console!")
 
-@app.route('/')
+@app.route("/")
 def index():
-    return "Hello, web!"
+    form = LinkForm()
+    links = Link.query.order_by(Link.link_id.desc()).all()
+    return render_template("index.html", form=form, links=links)
 
-app.run(host='0.0.0.0')
\ No newline at end of file
+
+@app.route("/add", methods=["POST"])
+def add():
+    form = LinkForm()
+    if form.validate_on_submit():
+        chars = string.digits + string.ascii_letters
+        
+        while (True):
+            short_url = ''.join(random.choice(chars) for i in range(3))
+            link_exists = Link.query.filter_by(short_url=short_url).first()
+
+            if not link_exists:
+                break
+
+        link = Link(long_url=form.long_url.data, short_url=short_url)
```

Comment:
You may want to perform some validation on the URL? For example, you could ask Python to try to parse it.

Reply:
URL validation will be added in the future